macos/package_files/3.10.0/postinstall.sh

```diff
@@ -14,6 +14,12 @@ DIR="/Library/Ossec"
 INSTALLATION_SCRIPTS_DIR="${DIR}/packages_files/agent_installation_scripts"
 SCA_FILES_DIR="${INSTALLATION_SCRIPTS_DIR}/sca"
 
+if [ `launchctl getenv WAZUH_PKG_UPGRADE` == true ]; then
```

Comment:
This notation is very legacy. Change it to `$(launchtl ...)`. In addition, you must use a single `=` instead of `==`.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/221/commits/3a2266309751f5e616ee1e44cf19e96350617fa9 and https://github.com/wazuh/wazuh-packages/pull/221/commits/1cb5ebf17dba1bb08ef9d4b8eee71a706945d8ac